### PR TITLE
Update examples to 1.13.x standards

### PR DESCRIPTION
**Pull request by typeoneerror**
* No Ember.View
* No Ember.keys

Example more es6-y.

### CODE REVIEW

File: source/templates/binding-element-attributes.md [hunk 1]
```diff
@@ -79,23 +79,19 @@ Now the same handlebars code above renders the following HTML:
...
-    this._super();
-    var self = this;
-
-    // bind attributes beginning with 'data-'
-    Ember.keys(this).forEach(function(key) {
-      if (key.substr(0, 5) === 'data-') {
-        self.get('attributeBindings').pushObject(key);
+export default Ember.Component.reopen({
+  bindAttributes: Ember.on('init', function() {
+    Object.keys(this).forEach((key) => {
```
Comment (locks) [inline source/templates/binding-element-attributes.md]: unsure about `Object.keys`

**Comment by locks** [comment]
I could swear a previous PR removed all mentions of `Ember.View`. Maybe because the PR is pointing to the `v1.13.0` branch, not master. Can you update?

File: source/templates/binding-element-attributes.md (hunk 1 above)
Comment (typeoneerror) [inline source/templates/binding-element-attributes.md]: I am too. Which seems like that might be a problem. I get a deprecation warning if I use Ember.keys with 1.13.x, and others are removing it (https://github.com/switchfly/ember-test-helpers/pull/71), so seemed right? Unsure.

**Comment by typeoneerror** [comment]
Still there in master: 

* https://github.com/emberjs/guides/blob/master/source/object-model/bindings.md.
* https://github.com/emberjs/guides/blob/master/source/templates/binding-element-attributes.md

I'll move the PR to master.

File: source/templates/binding-element-attributes.md (hunk 1 above)
Comment (locks) [inline source/templates/binding-element-attributes.md]: @typeoneerror according to the documentation "On browsers that support it, this uses the native Object.keys implementation." so I think you are right. I thought maybe it did something with computed properties or the like.

Comment (typeoneerror) [inline source/templates/binding-element-attributes.md]: @locks the open PR has been reverted back to Ember.keys. see: https://twitter.com/trek/status/628305995589152768.

Comment (locks) [inline source/templates/binding-element-attributes.md]: :+1: thank you for the clarification

**Pull request closed without merging**